content/test/integration/E2ETestEnvironment.ts

```diff
@@ -203,9 +203,7 @@ export function loadTestEnvironment(
     describe(name, () => {
       const testEnv = new E2ETestEnvironment()
 
-      it('starts the test environment', async () => {
-        await testEnv.start(overrideConfigs)
-      })
+      beforeEach(async () => { await testEnv.start(overrideConfigs) })
```

Comment:
Please consider reverting this behavior. Granularity of tests should be executed at "describe" level in stateful flow based tests.

Reply:
I read more about Jest and inspected the involved code and i think you're right, it doesn't feel correct to use `afterEach/beforeEach`. On the other hand, i found that using `beforeAll/afterAll` would work in the same way as using `it` but also would give use the chance to run a test singularly which i think is really useful for developing/debugging. Do you have any concerns about using `beforeAll/afterAll`?

Reply:
A bit of story: Jasmine errors in `beforeAll/afterAll` were sent to /dev/null, yielding tests invalid with no information. That is the reason why `beforeAll/afterAll` were converted to `it`.

I think that if Jest does the same, we may keep them as `it` (Jest embeeds Jasmine, ouch). But I'd give it a try converting them to `beforeAll/afterAll` and putting a `throw` inside them to see what happens. If the suite fails then we are OK.

If errors are sent to /dev/null as well, I think keeping `it` is a best decision

Reply:
Oh okay i see. As i read in some github issues some time ago that was an issue in Jest too (as it used jasmine in the darks), but now in `v27` jasmine was replaced by `jest-circus` and that issue was fixed, they say.
Anyway, i tested it myself and throwing an error within `beforeAll/afterAll` makes the test suite to fail and reports the errors correctly. So i think we're safe.

Reply:
Ok, I think it is safe to change to `beforeAll/afterAll` then

Reply:
sounds good